homeassistant/components/simplisafe/lock.py

```diff
@@ -75,9 +74,9 @@ def async_update_from_rest_api(self) -> None:
         self._attr_extra_state_attributes.update(
             {
                 ATTR_LOCK_LOW_BATTERY: self._lock.lock_low_battery,
-                ATTR_JAMMED: self._lock.state == LockStates.jammed,
```

Comment:
This is a breaking change, right? The jammed state attribute was removed and replaced by returning jammed as state.

Reply:
Good call, @MartinHjelmare. I've added that info to the PR description.